setup.py

```diff
@@ -0,0 +1,38 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="metrixpp",
+    version="0.0.1a",
```

Comment:
When ready, it is better to continue from the version we had.. Feel free to make a release via the next github tag..

Reply:
As soon as this is on master I will create a release and set the packet version to the new release version (I would suggest 1.6.0)